routes/.eslintrc.json

```diff
@@ -0,0 +1,6 @@
+{
+  "rules": {
+    "no-param-reassign": "off"
```

Comment:
I think this is a bad idea, for example with this rule we found a better solution to use to use `findByIdAndUpdate`